### ISSUE TITLE
Search page - post a guide request

### ISSUE THREAD

**Issue by sophiakc** [openfarmcc/OpenFarm#922]
Hi @roryaronson @simonv3  and all

On the [search page](https://github.com/openfarmcc/OpenFarm/issues/874), I would add the possibility to create a guide request through one of the thumbnail. Similar to Craigslist, someone looking to grow a plant in a certain way (and possibly not finding it yet on Openfarm), would be able to post a request in the form of a guide. This would encourage interactions between authors and readers.
It could be systematically shown for now to see how people react to it. Or shown only if 0 guide based on the search.

If you're good with that, I can update the wireframe. What do you think?

**Comment by simonv3** [comment]
I like it!

So we'd track the requests for growing on a crop, I assume? That should be fairly doable.

What happens with the number of requests if a guide for that crop gets created? Do we set it back to zero?

Going further down the rabbit hole - do we track specifics about the request? Like a location and an environment? Simplest of course is keeping track just that someone requested a guide, but if we'll track details in the future that might be good to know before implementation.

**Comment by sophiakc** [comment]
@simonv3 

> So we'd track the requests for growing on a crop, I assume? That should be fairly doable.
Yes

> What happens with the number of requests if a guide for that crop gets created? Do we set it back to zero?
First, we want to see if this request feature is used.
Second and theoretically, the number and the requests will stay the same. If a guide has been created and is answering the request, it would be added as a comment to the request. And possibily similar requests could be linked together on the comment section.
So that people finding one request could see the other ones and guide answer.


> Going further down the rabbit hole - do we track specifics about the request? Like a location and an environment? Simplest of course is keeping track just that someone requested a guide, but if we'll track details in the future that might be good to know before implementation.
Basically, a request would borrow the same template than a guide editing template, so yes for the specifics: like location, time of the year, crop. Up to which the person is willing to complete it.

**Comment by sophiakc** [comment]
@simonv3 @roryaronson & all
Here a suggestion of wireframes for the Search page 
- empty state
- result state with two additional call to actions: create a guide, post a request (this will open the same template "create a guide", the only difference being one identified as guide, the other as request)

![wireframe - empty state](https://cloud.githubusercontent.com/assets/3863675/24132756/0beecb84-0db5-11e7-85df-6993665c7c5f.jpg)

![wireframe - live](https://cloud.githubusercontent.com/assets/3863675/24132768/23d8e266-0db5-11e7-8b56-bfcbaebf4e55.jpg)

If your good with that, I'll update this issue: https://github.com/openfarmcc/OpenFarm/issues/874
